run_scripts/setup_ubuntu_vm.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install Python 3.6
+sudo apt update && sudo apt install -y python3.6 python3-pip git
+sudo -H pip3 install pipenv
+
+# Install Docker
+sudo apt install -y \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    gnupg-agent \
+    software-properties-common
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
Is there a need to tag a version of docker we want?

Reply:
added versions